src/core/annotation.js

```diff
@@ -439,13 +439,40 @@ class Annotation {
     );
   }
 
-  isHidden(annotationStorage) {
+  /**
+   * Check if the annotation must be printed in taking into account
+   * the value found in the annotationStorage which may have been set
+   * through JS.
+   *
+   * @public
+   * @memberof Annotation
+   * @param {AnnotationStorage} [annotationStorage] - Storage for annotation
+   */
+  mustBePrinted(annotationStorage) {
```

Comment:
Nit: In the existing code, all "view" methods/getters always occur *before* the "print" ones.
Can we please re-order these two new methods similarity, such that `mustBeViewed` is place before `mustBePrinted` for consistency?